pandunia-loge.csv

```diff
@@ -522,7 +522,7 @@ Burkina Faso|desha|BF||Burkina Faso||Burkina Faso||||||||||||||Burkina Faso|Burk
 bus|||deu:fra:eng:spa:bus, rus:автобус (avtobus), hin:बस (bas), urd:(bas), jpn:バス (basu), kor:버스 (beoseu), fas:(otobus), tur:otobüs, swa:basi, zho:巴士 (bāshì)|bus||bus (autobús)|||||||||||||buso|linja-auto (bussi)|autobus, autokar
 buste|||fra:buste, eng:bust, spa:por:busto, rus:бюст (byust), deu:Büste|bust (bosom)||pecho (busto)||||||||||||||povi|biust
 buta|||eng:boot, fra:botte, spa:por:bota, rus:ботинок (botinok), pol:may:but, fas: بوت‎ (but), jpn:ブーツ (būtsu), swa:buti|boot|botte|bota|bota|ботинок|||ブーツ|||||||||saapas (buutsi)|but
-Lung Desha|desha|BT||Bhutan||Bután||||||||||||||Bhutan|Bhutan
+Drukyul|desha|BT||Bhutan||Bután||||||||||||||Bhutan|Bhutan
```

Comment:
I got the impression that this Romanization is based on outdated Dzongkha orthography and the pronunciation has changed. The country name is pronounced something like "Du Ü". See https://en.wiktionary.org/wiki/%E0%BD%A0%E0%BD%96%E0%BE%B2%E0%BD%B4%E0%BD%82%E0%BC%8B%E0%BD%A1%E0%BD%B4%E0%BD%A3#Dzongkha

The origin of Druk/Drug/Brug is the same as the origin of Mandarin lóng, which was in Old Chinese b-rong or m-rong, and in Proto-Sino-Tibetan *m-bru(ŋ/k). See https://en.wiktionary.org/wiki/%E9%BE%8D#Pronunciation
That's why 'Lung Desha' is etymologically good although the Sinitic and Tibetan branches have diverged far apart.

Reply:
hm, I think you're rite.  I assumed <dr> was how they romanized <ɖ>, but it looks like it's actually a combination of /b/ and the /r/ diacritick.  all things considerd, **lung** is probably about as close to /ʈu/ as **druk** or **brug**.